cyclictest/cmd.sh

```diff
@@ -77,22 +86,32 @@ if [[ "$stress" == "true" ]]; then
     done
 fi
 
-cyccore=${cpus[0]}
-cindex=1
+cyccore=${cpus[1]}
+cindex=2
 ccount=1
 while (( $cindex < ${#cpus[@]} )); do
-	cyccore="${cyccore},${cpus[$cindex]}"
-	cindex=$(($cindex + 1))
-        ccount=$(($ccount + 1))
+    cyccore="${cyccore},${cpus[$cindex]}"
+    cindex=$(($cindex + 1))
+    ccount=$(($ccount + 1))
 done
 
+sibling=`cat /sys/devices/system/cpu/cpu${cpus[0]}/topology/thread_siblings_list | awk -F '[-,]' '{print $2}'`
```

Comment:
this assumes that hyper threading is enabled; the case that hyper threading is enabled should be considered as well

Reply:
I copied this directly from the oslat script. I think this is OK if hyper threading is disabled - in that case the thread_siblings_list will only contain a single cpu - for example:
`/sys/devices/system/cpu/cpu0/topology/thread_siblings_list:0`
So $sibling will be and empty string and the "if" on line 99 will not find a match, so nothing will be removed from the cyccore list.